### ISSUE TITLE
Cannot login to nc after activating static ip address

### ISSUE THREAD

**Issue by jpilgrim** [nextcloud/nextcloudpi#1733]
I have created a new SSD card from  `NextCloudPi_RaspberryPi_v1.51.0.img`.
I was able to log in to nc and NextCloudPi

After booting the PI and activating nc/ncp, I changed the *Set up a static IP address* to static in NextCloudPi panel.
After that, I was not able to login to nc anymore. Changing the password with nc-admin did not help.

Actually, this is my second attempt. In the first attempt, I had the same problem but a message to change "trusted_domains" in config.php. This message did not appear in the second attempt, and the static ip is already listed there.

So, how to reproduce: 
1. Create ssd from NextCloudPi_RaspberryPi_v1.51.0.img`
2. Start system, activate etc.
3. Set up a static IP address to some static IP

<details>
<summary>Set static IP address output</summary>
```
[ nc-static-IP ] (Tue Feb 14 21:17:31 GMT 2023)
System config value trusted_domains => 1 set to string 192.168.178.200
warning: No domain found. Defaulting to 'nextcloudpi'
System config value trusted_domains => 3 set to string nextcloudpi
System config value overwrite.cli.url set to string https://nextcloudpi/
System config value trusted_proxies => 11 set to string 127.0.0.1
System config value trusted_proxies => 12 set to string ::1
System config value trusted_proxies => 13 set to string nextcloudpi
System config value trusted_proxies => 14 set to string 192.168.178.28
192.168.178.200
Setup notify_push (attempt 1/5)
✓ redis is configured
✓ push server is receiving redis messages
✓ push server is receiving redis messages
✓ push server can load mount info from database
✓ push server can connect to the Nextcloud server
✓ push server is a trusted proxy
✓ push server is running the same version as the app
configuration saved
Static IP set to 192.168.178.200
```
</details>

**System information**

<details>
<summary>NextcloudPi diagnostics</summary>

```

NextcloudPi version  v1.51.0
NextcloudPi image    NextCloudPi_RaspberryPi_v1.51.0.img
OS                   Debian GNU/Linux 11. 5.15.61-v8+ (aarch64)
automount            no
USB devices          none
datadir              /opt/ncdata/data
data in SD           yes
data filesystem      ext2/ext3
data disk usage      2.5G/59G
rootfs usage         2.5G/59G
swapfile             /var/swap
dbdir                /var/lib/mysql
Nextcloud check      ok
Nextcloud version    25.0.3.2
HTTPD service        up
PHP service          up
MariaDB service      up
Redis service        up
HPB service          up
Postfix service      up
Internet check       ok
Public IPv4          ***REMOVED SENSITIVE VALUE***
Public IPv6          ***REMOVED SENSITIVE VALUE***
Port 80              closed
Port 443             closed
IP                   ***REMOVED SENSITIVE VALUE***
Gateway              ***REMOVED SENSITIVE VALUE***
Interface            eth0
Certificates         ***REMOVED SENSITIVE VALUE***
NAT loopback         no
Uptime               27min

```
</details>

<details>
<summary>Nextcloud configuration</summary>

```

{
    "system": {
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": {
            "0": "localhost",
            "7": "nextcloudpi",
            "5": "nextcloudpi.local",
            "8": "nextcloudpi.lan",
            "3": "nextcloudpi",
            "11": "77.188.83.205",
            "1": "192.168.178.200",
            "14": "nextcloudpi"
        },
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "mysql",
        "version": "25.0.3.2",
        "overwrite.cli.url": "https:\/\/nextcloudpi\/",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "mysql.utf8mb4": true,
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "memcache.local": "\\OC\\Memcache\\Redis",
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "port": 0,
            "timeout": 0,
            "password": "***REMOVED SENSITIVE VALUE***"
        },
        "tempdirectory": "\/opt\/ncdata\/data\/tmp",
        "mail_smtpmode": "sendmail",
        "mail_smtpauthtype": "LOGIN",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "preview_max_x": "2048",
        "preview_max_y": "2048",
        "jpeg_quality": "60",
        "overwriteprotocol": "https",
        "maintenance": false,
        "logfile": "\/opt\/ncdata\/data\/nextcloud.log",
        "trusted_proxies": "***REMOVED SENSITIVE VALUE***"
    }
}

```
</details>

<details>
<summary>HTTPd logs</summary>

```

[Thu Sep 22 04:06:18.736155 2022] [ssl:warn] [pid 531:tid 547843224640] AH01909: localhost:443:0 server certificate does NOT include an ID which matches the server name
[Thu Sep 22 04:06:18.736842 2022] [ssl:error] [pid 531:tid 547843224640] AH02217: ssl_stapling_init_cert: can't retrieve issuer certificate! [subject: CN=fv-az351-756 / issuer: CN=fv-az351-756 / serial: 749A9144E7E4C538C232332718BA1E03BD5CBD82 / notbefore: Jan 22 01:35:25 2023 GMT / notafter: Jan 19 01:35:25 2033 GMT]
[Thu Sep 22 04:06:18.736878 2022] [ssl:error] [pid 531:tid 547843224640] AH02604: Unable to configure certificate localhost:443:0 for stapling
[Thu Sep 22 04:06:18.895825 2022] [ssl:warn] [pid 684:tid 547843224640] AH01909: localhost:443:0 server certificate does NOT include an ID which matches the server name
[Thu Sep 22 04:06:18.896254 2022] [ssl:error] [pid 684:tid 547843224640] AH02217: ssl_stapling_init_cert: can't retrieve issuer certificate! [subject: CN=fv-az351-756 / issuer: CN=fv-az351-756 / serial: 749A9144E7E4C538C232332718BA1E03BD5CBD82 / notbefore: Jan 22 01:35:25 2023 GMT / notafter: Jan 19 01:35:25 2033 GMT]
[Thu Sep 22 04:06:18.896297 2022] [ssl:error] [pid 684:tid 547843224640] AH02604: Unable to configure certificate localhost:443:0 for stapling
[Thu Sep 22 04:06:18.919969 2022] [mpm_event:notice] [pid 684:tid 547843224640] AH00489: Apache/2.4.54 (Debian) OpenSSL/1.1.1n configured -- resuming normal operations
[Thu Sep 22 04:06:18.920151 2022] [core:notice] [pid 684:tid 547843224640] AH00094: Command line: '/usr/sbin/apache2'
[Tue Feb 14 20:59:32.587743 2023] [proxy_fcgi:error] [pid 685:tid 547665727872] [client ::1:39400] AH01067: Failed to read FastCGI header
[Tue Feb 14 20:59:32.587910 2023] [proxy_fcgi:error] [pid 685:tid 547665727872] (104)Connection reset by peer: [client ::1:39400] AH01075: Error dispatching request to : 
[Tue Feb 14 21:01:29.830085 2023] [mpm_event:notice] [pid 684:tid 547843224640] AH00493: SIGUSR1 received.  Doing graceful restart
[Tue Feb 14 21:01:29.866621 2023] [ssl:warn] [pid 684:tid 547843224640] AH01909: localhost:4443:0 server certificate does NOT include an ID which matches the server name
[Tue Feb 14 21:01:29.867013 2023] [ssl:error] [pid 684:tid 547843224640] AH02217: ssl_stapling_init_cert: can't retrieve issuer certificate! [subject: CN=fv-az351-756 / issuer: CN=fv-az351-756 / serial: 749A9144E7E4C538C232332718BA1E03BD5CBD82 / notbefore: Jan 22 01:35:25 2023 GMT / notafter: Jan 19 01:35:25 2033 GMT]
[Tue Feb 14 21:01:29.867054 2023] [ssl:error] [pid 684:tid 547843224640] AH02604: Unable to configure certificate localhost:4443:0 for stapling
[Tue Feb 14 21:01:29.871067 2023] [mpm_event:notice] [pid 684:tid 547843224640] AH00489: Apache/2.4.54 (Debian) OpenSSL/1.1.1n configured -- resuming normal operations
[Tue Feb 14 21:01:29.871149 2023] [core:notice] [pid 684:tid 547843224640] AH00094: Command line: '/usr/sbin/apache2'

```
</details>

<details>
<summary>Database logs</summary>

```

tail: cannot open '/var/log/mysql/*.log' for reading: No such file or directory

```
</details>

<details>
<summary>Nextcloud logs</summary>

```

{"reqId":"jCKHe6O8u4wnpTCCkYd3","level":3,"time":"2023-01-22T01:55:01+00:00","remoteAddr":"","user":"--","app":"cron","method":"","url":"--","message":"Not installed","userAgent":"--","version":"","exception":{"Exception":"Exception","Message":"Not installed","Code":0,"Trace":[{"file":"/var/www/nextcloud/lib/base.php","line":680,"function":"checkInstalled","class":"OC","type":"::"},{"file":"/var/www/nextcloud/lib/base.php","line":1144,"function":"init","class":"OC","type":"::"},{"file":"/var/www/nextcloud/cron.php","line":43,"args":["/var/www/nextcloud/lib/base.php"],"function":"require_once"}],"File":"/var/www/nextcloud/lib/base.php","Line":284,"CustomMessage":"--"}}
{"reqId":"hietpzdjSBJdaxhEho5e","level":3,"time":"2023-01-22T01:57:04+00:00","remoteAddr":"","user":"ncp","app":"PHP","method":"","url":"--","message":"exif_read_data(): File not supported at /var/www/nextcloud/lib/private/Metadata/Provider/ExifProvider.php#59","userAgent":"--","version":"25.0.3.2","exception":{"Exception":"Error","Message":"exif_read_data(): File not supported at /var/www/nextcloud/lib/private/Metadata/Provider/ExifProvider.php#59","Code":0,"Trace":[{"function":"onError","class":"OC\\Log\\ErrorHandler","type":"::"},{"file":"/var/www/nextcloud/lib/private/Metadata/Provider/ExifProvider.php","line":59,"function":"exif_read_data"},{"file":"/var/www/nextcloud/lib/private/Metadata/MetadataManager.php","line":68,"function":"execute","class":"OC\\Metadata\\Provider\\ExifProvider","type":"->"},{"file":"/var/www/nextcloud/lib/private/Metadata/FileEventListener.php","line":101,"function":"generateMetadata","class":"OC\\Metadata\\MetadataManager","type":"->"},{"file":"/var/www/nextcloud/lib/private/EventDispatcher/ServiceEventListener.php","line":87,"function":"handle","class":"OC\\Metadata\\FileEventListener","type":"->"},{"file":"/var/www/nextcloud/3rdparty/symfony/event-dispatcher/EventDispatcher.php","line":251,"function":"__invoke","class":"OC\\EventDispatcher\\ServiceEventListener","type":"->"},{"file":"/var/www/nextcloud/3rdparty/symfony/event-dispatcher/EventDispatcher.php","line":73,"function":"callListeners","class":"Symfony\\Component\\EventDispatcher\\EventDispatcher","type":"->"},{"file":"/var/www/nextcloud/lib/private/EventDispatcher/EventDispatcher.php","line":88,"function":"dispatch","class":"Symfony\\Component\\EventDispatcher\\EventDispatcher","type":"->"},{"file":"/var/www/nextcloud/lib/private/EventDispatcher/EventDispatcher.php","line":100,"function":"dispatch","class":"OC\\EventDispatcher\\EventDispatcher","type":"->"},{"file":"/var/www/nextcloud/lib/private/Files/Node/HookConnector.php","line":118,"function":"dispatchTyped","class":"OC\\EventDispatcher\\EventDispatcher","type":"->"},{"file":"/var/www/nextcloud/lib/private/legacy/OC_Hook.php","line":106,"function":"postWrite","class":"OC\\Files\\Node\\HookConnector","type":"->"},{"file":"/var/www/nextcloud/lib/private/Files/View.php","line":641,"function":"emit","class":"OC_Hook","type":"::"},{"file":"/var/www/nextcloud/lib/private/Files/View.php","line":692,"function":"emit_file_hooks_post","class":"OC\\Files\\View","type":"->"},{"file":"/var/www/nextcloud/lib/private/Files/Node/File.php","line":71,"function":"file_put_contents","class":"OC\\Files\\View","type":"->"},{"file":"/var/www/nextcloud/lib/private/legacy/OC_Util.php","line":257,"function":"putContent","class":"OC\\Files\\Node\\File","type":"->"},{"file":"/var/www/nextcloud/lib/private/legacy/OC_Util.php","line":216,"function":"copyr","class":"OC_Util","type":"::"},{"file":"/var/www/nextcloud/lib/private/User/Session.php","line":553,"function":"copySkeleton","class":"OC_Util","type":"::"},{"file":"/var/www/nextcloud/lib/private/User/Session.php","line":403,"function":"prepareUserLogin","class":"OC\\User\\Session","type":"->"},{"file":"/var/www/nextcloud/lib/private/User/Session.php","line":616,"function":"completeLogin","class":"OC\\User\\Session","type":"->","args":["*** sensitive parameters replaced ***"]},{"file":"/var/www/nextcloud/lib/private/User/Session.php","line":355,"function":"loginWithPassword","class":"OC\\User\\Session","type":"->","args":["*** sensitive parameters replaced ***"]},{"file":"/var/www/nextcloud/lib/private/Setup.php","line":433,"function":"login","class":"OC\\User\\Session","type":"->","args":["*** sensitive parameters replaced ***"]},{"file":"/var/www/nextcloud/core/Command/Maintenance/Install.php","line":105,"function":"install","class":"OC\\Setup","type":"->","args":["*** sensitive parameters replaced ***"]},{"file":"/var/www/nextcloud/3rdparty/symfony/console/Command/Command.php","line":255,"function":"execute","class":"OC\\Core\\Command\\Maintenance\\Install","type":"->"},{"file":"/var/www/nextcloud/3rdparty/symfony/console/Application.php","line":1009,"function":"run","class":"Symfony\\Component\\Console\\Command\\Command","type":"->"},{"file":"/var/www/nextcloud/3rdparty/symfony/console/Application.php","line":273,"function":"doRunCommand","class":"Symfony\\Component\\Console\\Application","type":"->"},{"file":"/var/www/nextcloud/3rdparty/symfony/console/Application.php","line":149,"function":"doRun","class":"Symfony\\Component\\Console\\Application","type":"->"},{"file":"/var/www/nextcloud/lib/private/Console/Application.php","line":213,"function":"run","class":"Symfony\\Component\\Console\\Application","type":"->"},{"file":"/var/www/nextcloud/console.php","line":100,"function":"run","class":"OC\\Console\\Application","type":"->"},{"file":"/var/www/nextcloud/occ","line":11,"args":["/var/www/nextcloud/console.php"],"function":"require_once"}],"File":"/var/www/nextcloud/lib/private/Log/ErrorHandler.php","Line":92,"CustomMessage":"--"}}
{"reqId":"dUder3Mi7mHMJL9PvHBg","level":2,"time":"2023-01-22T01:59:55+00:00","remoteAddr":"","user":"--","app":"no app in context","method":"","url":"--","message":"Temporary directory /var/www/nextcloud/data/tmp is not present or writable","userAgent":"--","version":"25.0.3.2","data":[]}
{"reqId":"dUder3Mi7mHMJL9PvHBg","level":2,"time":"2023-01-22T01:59:55+00:00","remoteAddr":"","user":"--","app":"no app in context","method":"","url":"--","message":"Temporary directory /var/www/nextcloud/data/tmp is not present or writable","userAgent":"--","version":"25.0.3.2","data":[]}
{"reqId":"fIemf2PKbuxhXOBpdDae","level":2,"time":"2023-01-22T01:59:58+00:00","remoteAddr":"","user":"--","app":"no app in context","method":"","url":"--","message":"Temporary directory /var/www/nextcloud/data/tmp is not present or writable","userAgent":"--","version":"25.0.3.2","data":[]}
{"reqId":"fIemf2PKbuxhXOBpdDae","level":2,"time":"2023-01-22T01:59:58+00:00","remoteAddr":"","user":"--","app":"no app in context","method":"","url":"--","message":"Temporary directory /var/www/nextcloud/data/tmp is not present or writable","userAgent":"--","version":"25.0.3.2","data":[]}

```
</details>

**Comment by OfficialMuffin** [comment]
Can you not connect to the new IP at all? What exactly happens when you type in `https://192.168.178.200`?

**Comment by jpilgrim** [comment]
I can connect to the new IP via SSH and via the NectCloutPi Panel. The only thing that doesn't work anymore is log into nc.

**Comment by OfficialMuffin** [comment]
What's the browser error when attempting to connect to nc?

**Comment by jpilgrim** [comment]
There is no browser error, I just cannot login nextcloud.

**Comment by jpilgrim** [comment]
Forget it. 

I tried again, this time setting up Debian manually (starting from the Raspberry Pi OS image created via Raspberry Pi Imager). I had configured the static IP manually. Then installed NextcloudPi via the curl-script. (oh, fixed some locale problem before...).

But I'm deleting that NextCloudPi installation now. Bugs over bugs. Starts with missing logo (which does not really build up trust) and ends with a corrupt calendar application which is a show stopper: I imported some calendar data, and it looked nice -- but most entries were gone after login/logout. And it is super slow (compared to a previous version I had installed manually which was running nicely on my Pi 3 Model B+. That one uses nginx instead of Apache -- although I'm not quite sure whether that is the problem).

Will give it another try in a year ;-)
